webroot/js/snodas_map.js

```diff
@@ -230,14 +232,44 @@ query_selector.onclick = function(event) {
     query_selector.select_query_by_element(event.target);
 }
 
-var pp_table_html = '<table class="table table-borderless mb-3 border" id="snodas-pourpoint-table"><tbody><tr><th scope="row">AWDB ID</th><td id="snodas-pourpoint-awdb-id"></td></tr><tr><th scope="row">Name</th><td id="snodas-pourpoint-name"></td></tr></tbody></table>';
+var awdb_select = '<tr><th>Search</th><td><input type="search" id="awdb-filter" autocomplete="off"></td></tr><tr><th scope="row" colSpan="2"><select name="awdb-select" id="awdb-select"></select></th></tr>'
```

Comment:
I think you need to specify some CSS for the select box, perhaps

```css
select#awdb-select {
    width: 100%;
    overflow: hidden;
    text-overflow: ellipsis;
    white-space: nowrap;
}
```

Reply:
It also seems that if a user enters in some search term the top result is not selected, and if it is what they want they can't just click it. I think ideally the top result would be autoselected, but I couldn't figure out a quick way to do that. But I did find a way to click on an item that was already selected and have it fire the change event.

```suggestion
var awdb_select = '<tr><th>Search</th><td><input type="search" id="awdb-filter" autocomplete="off"></td></tr><tr><th scope="row" colSpan="2"><select name="awdb-select" id="awdb-select" onchange="selectMapLyr();" onfocus="this.selectedIndex = -1;"></select></th></tr>'
```

With this you can remove the `change` event listener, I believe.

Reply:
I'm not sure about what the auto-selection is actually doing based on the description, but it could prevent the user from keeping the previously selected forecast station in the UI. We want to keep the previously selected forecast station unchanged (see the image below) until a list item is selected by the user.

![image](https://github.com/PSU-CSAR/snodas-ui/assets/11510708/34de9974-4181-427f-842c-9912b280a3f3)

Reply:
I don't know why setting the selectedIndex to -1 works, but it does. I just tested this and it seems to work as desired. The AWDB ID and Name text fields persist unless a different AOI is selected either from the select list or from the map. @jdduh if this isn't working as desired, please provide the steps for the use case that is failing.